### ISSUE TITLE
Standardise connector names

### ISSUE THREAD

**Issue by wendyOn** [infinyon/fluvio-connectors#153]
- http conflicts with http crate have to use ::http
- if we decide to release to crates.io helps later
- e.g. http -> fluvio-connector-http

**Comment by morenol** [comment]
I think that we should use the same name that we use in https://hub.docker.com

https://hub.docker.com/u/infinyon

**Comment by pinkforest** [comment]
Good catch yeah fluvio-connect-xxxx sounds reasonable to align with docker hub.
